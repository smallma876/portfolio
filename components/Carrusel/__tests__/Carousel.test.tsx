import { render, screen } from "@testing-library/react";
import Carousel from "../Carousel";

describe("Carousel test", () => {
  it("Render page carousel", () => {
    const { container } = render(<Carousel />);
    expect(screen.getByText("Carousel")).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
