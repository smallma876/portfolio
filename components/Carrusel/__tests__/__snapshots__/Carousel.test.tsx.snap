// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel test Render page carousel 1`] = `
<div>
  <div>
    Carousel
  </div>
</div>
`;
